tests/Feature/TrafficAnalytics/ApplicationTrafficOverviewTest.php

```
@@ -0,0 +1,131 @@
+
+        return '{}';
+    }
+}
+
+// #[Lazy] components render a placeholder first; trigger the deferred mount as the
+// browser would via the x-intersect __lazyLoad call, then continue asserting.
+function loadLazy(Testable $component): Testable
+{
+    preg_match('/__lazyLoad\(&#039;([^&]+)&#039;\)/', $component->html(), $matches);
+
+    return $component->call('__lazyLoad', $matches[1]);
```

PHPStan (2.2.7)

[warning] Offset 1 might not exist on array{}|array{non-falsy-string, non-empty-string}.


(offsetAccess.notFound)